freqtrade/persistence/models.py

```diff
@@ -324,7 +324,7 @@ def to_json(self) -> Dict[str, Any]:
             'profit_pct': round(self.close_profit * 100, 2) if self.close_profit else None,
             'profit_abs': self.close_profit_abs,
 
-            'sell_reason': self.sell_reason,
+            'sell_reason': str(self.sell_reason) + (f' ({self.buy_tag})' if self.buy_tag else ''),
```

Comment:
Definitely don't do this. 
This decision should be left to the consuming party (it get's both informations independently) - not forced from the bot.